### ISSUE TITLE
sitewide, design system, CMS — 508-defect-2 [AXE-CORE]: Heading levels should increase by one - footer issue

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7755]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team & project: vsa-public-websites, AMA follow-on
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Headings on a web page should progress, increasing by one, in sections of content. The headings in the footer are currently tagged as `<h4>`. On pages that don't have an `<h3>`, this results in an accessibility issue. 

Moderate Template Issue | Heading levels should only increase by one
`<h4 class="va-footer-linkgroup-title>Veteran programs and services</h4>`

1. After you request a decision review — https://preview-prod.vfs.va.gov/preview?nodeId=3008
1. Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011
1. Higher-Level Reviews — https://preview-prod.vfs.va.gov/preview?nodeId=3003
1. Board Appeals — https://preview-prod.vfs.va.gov/preview?nodeId=3029
1. Contested Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3030
1. Insurance claims — https://preview-prod.vfs.va.gov/preview?nodeId=3025
1. FAQs — https://preview-prod.vfs.va.gov/preview?nodeId=3004
1. Get help with review request — https://preview-prod.vfs.va.gov/preview?nodeId=3009

### Why it Matters
The underlying purpose of headers is to convey the structure of the page. For sighted users, the same purpose is achieved using different sizes of text. Text size, however, is not helpful for users of screen readers, because a screen reader identifies a header only if it is properly marked-up. When heading elements are applied correctly, the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements applied in hierarchical order. All of these qualities make headings valuable tools for screen reader users. Similar to the way sighted users can glance at a page and get a sense of its contents, screen reader users can navigate through headings. Well written and properly ordered headings can save screen reader time and frustration.

In addition to making the page more accessible, headings have other benefits since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging & production

## Steps to Recreate

1. Enter any of the above urls in browser, in the VA environment
1. Have developer tools open, and the axe browser extension loaded
1. Enter Urgent care for facility type and Community urgent care providers for service type 
1. Run an axe audit
1. Verify that there is an error of "Heading levels should only increase by one"

## Possible Fixes (optional)

Change the structure of the footer info lists to use a `<dl>` or change the `<h4>` to an `<h2>`.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)
* [MDN dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)
* [MDN ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role)
* [MDN Description List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

**Comment by 1Copenut** [comment]
This has been fixed at some point, so I"m going to close it.
